extensions/amp-fit-text/0.1/test/test-amp-fit-text.js

```diff
@@ -71,6 +70,39 @@ describes.realWin(
       await ft.implementation_.mutateElement(() => {});
       const content = ft.querySelector('.i-amphtml-fit-text-content');
       expect(content.textContent).to.equal(newText);
+      await ft.implementation_.unlayoutCallback();
+    });
+
+    it('re-calculates font size if a resize is detected by the measurer', async () => {
+      const ft = await getFitText(
+        'Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque inermis reprehendunt.'
+      );
+      const updateFontSizeSpy = env.sandbox.spy(
+        ft.implementation_,
+        'updateFontSize_'
+      );
+      await new Promise((resolve) => {
+        setTimeout(() => {
+          resolve();
+        }, 90);
```

Comment:
Why `90`? Could be helpful to add an inline comment on why this value is meaningful.

Reply:
In this case `90` was arbitrarily chosen, i'll make it `100` and then add a comment to indicate why it was chosen.

Reply:
I wasn't sure if it was intentionally related to the throttle rate. Thanks for adding a comment :)

Reply:
The idea is to wait for the ResizeObserver to fire, but not until the next function call due to throttle. So any value less than 100 should be good.

Reply:
It turns out 100ms exactly is a border case which can pass but its not clear whats happening so I am gonna re-set it to 90 and adjust my comments accordingly.

Reply:
Oh got it, thanks both for clarifying!